src/NServiceBus.Msmq.AcceptanceTests/When_distributing_an_event.cs

```diff
@@ -81,7 +81,7 @@ public Subscriber1()
                 EndpointSetup<DefaultServer>(c =>
                 {
                     var transport = c.UseTransport<MsmqTransport>();
-                    transport.Routing().RegisterPublisherForType(PublisherEndpoint, typeof(MyEvent));
+                    transport.Routing().RegisterPublisherForType(typeof(MyEvent), PublisherEndpoint);
```

Comment:
Not related to this PR, but I wondered if we could automatically expose the endpoint name on the context by doing something like

context.EndpointNames[typeof(Publisher)]

and the convention calling is already done by the infra